ansible_collections/arista/avd/roles/eos_designs/python_modules/core_interfaces/utils.py

```diff
@@ -210,6 +212,7 @@ def _get_p2p_data(self, p2p_link: dict) -> dict:
             "peer_ip": ip[peer_index],
             "bgp_as": str(bgp_as[index]),
             "peer_bgp_as": str(bgp_as[peer_index]),
+            "description": descriptions[index] if descriptions[index] is not None else None,
```

Comment:
so ... :) this could just be `"description": descriptions[index]` right?

Reply:
haha, yes. Fixed.